plugins/amdgpu/amdgpu_plugin_drm.c

```diff
@@ -0,0 +1,66 @@
+#include <errno.h>
+#include <fcntl.h>
+#include <stdlib.h>
+#include <stdint.h>
+#include <stdbool.h>
+#include <string.h>
+#include <stdio.h>
+#include <unistd.h>
+#include <pthread.h>
+#include <semaphore.h>
+
+#include <sys/stat.h>
+#include <sys/mman.h>
+#include <sys/types.h>
+#include <sys/ioctl.h>
+#include <sys/syscall.h>
+#include <sys/sysmacros.h>
+#include <linux/limits.h>
+
+#include <dirent.h>
+#include "common/list.h"
+
+#include "criu-amdgpu.pb-c.h"
+
+#include <xf86drm.h>
+#include <libdrm/amdgpu.h>
+
+#include "xmalloc.h"
+#include "criu-log.h"
+#include "kfd_ioctl.h"
+#include "amdgpu_plugin_drm.h"
+#include "amdgpu_plugin_util.h"
+#include "amdgpu_plugin_topology.h"
+
+
+int amdgpu_plugin_drm_handle_device_vma(int fd, const struct stat *st)
+{
+	char path[PATH_MAX];
+	struct stat drm;
+	int ret = 0;
+
+	pr_info("Entered: %s\n", __func__);
```

Comment:
This looks like a debug message. Is it necessary?

Reply:
Removed